_projects/email-bridge.md

```diff
@@ -0,0 +1,60 @@
+---
+name: First-Class Email Bridge
+desc: Build a "gateway" email bridge to Matrix
+requirements:
+# Student requirements:
+ - Keen interest in bridging networks together
+difficulty: Hard
+issues:
+mentors:
+# First person in contact; mentors may change before project starts.
+ - Half-Shot
+initiatives:
+ - GSoC
+tags:
+ - Node.JS
+ - Typescript
+ - Bridges
+ - Email
+ - SMTP
+---
+
+Email is one of the most ubiqutous methods on the internet to communicate today, having
+been around for a very long time. It's been a dream for a long time that one day Matrix
+could be connected up to Email by translating incoming [SMTP](https://en.wikipedia.org/wiki/Simple_Mail_Transfer_Protocol)
+traffic to Matrix messages, and then bridging Matrix messages back into emails. 
+
+Matrix has a long history of being the best place to bridge different networks together. It
+has a [rich ecosystem of bridges](https://matrix.org/bridges/), making full use of the
+[Application Service API](https://matrix.org/docs/spec/application_service/r0.1.2). There
+are some Email bridges in existence already but here we are looking for someone to go
+above and beyond and build a native SMTP <-> Email bridge.
+
+### And this is where you come in
+
+We'd like you to build a Email bridge to Matrix. This would ideally take the form of a
+bridge process that handles it's own SMTP traffic rather than using an existing mail
+server, and would bridge messages to and from Matrix rooms based upon the room membership
+rather than any kind of storage in the bridge. This is typically what we call a gateway
+bridge where the user has to do no extra configuration to make rooms magically bridge!
```

Comment:
Unnecessary exclamation mark! :wink: 
```suggestion
bridge where the user has to do no extra configuration to make rooms magically bridge.
```

Reply:
I think we can afford a little excitement!